.gitignore

```diff
@@ -42,3 +42,29 @@ coverage.xml
 # Sphinx documentation
 docs/_build/
 docs/build
+
+# Os Specific
+.DS_STORE
+sequence_views.enaml
```

Comment:
Could you clean this up a bit ? ignoring .idea is fine but the new.enaml for example does not really makes sense.